modules/apiserversource-yaml.adoc

```diff
@@ -236,10 +273,34 @@ Data,
 [id="delete-apiserversource-yaml_{context}"]
```

Comment:
Same comment as for previous module--should Deleting the ApiServerSource be its own procedure module?